recipe/session/claims.js

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
```

Comment:
- I added exports in `session/index` I'm not sure if keeping both (session/claims) is still worth it. In general, I'd prefer the latter, but I see how it could cause problems when using the lib via script tags.
- With exports in `session/index` devs can simply use claims after adding the session recipe in a script tag like this:
```js
new supertokensSession.PrimitiveClaim({id: "ASdf", refresh: () => { console.log("refresh")}})
```